tasks/discover-granules/schemas/config.json

```diff
@@ -81,6 +81,11 @@
       "default": false,
       "type": "boolean"
     },
+    "httpRequestTimeout": {
+      "description": "Timeout (in seconds) for retreival by HTTP/HTTPS provider",
```

Comment:
```suggestion
      "description": "Timeout (in seconds) for retrieval by HTTP/HTTPS provider",
```

Reply:
Is this "retrieval" time, or "crawling" time? It's probably worth documenting the difference.